File: code/dotnet/sdk/src/Defaults.fs
module StarFederation.Datastar.Defaults

open System

let [<Literal>] Version = "1.0.0-beta.1"
let [<Literal>] VersionClientByteSize = 43972
let [<Literal>] VersionClientByteSizeGzip = 15021

let DefaultSettleDuration = TimeSpan.FromMilliseconds 300
let DefaultSseSendRetry = TimeSpan.FromMilliseconds 1000
let DefaultFragmentMergeMode = "morph"

File: code/php/sdk/src/Defaults.php
<?php
namespace starfederation\datastar;

use starfederation\datastar\enums\FragmentMergeMode;

class Defaults
{
    public const DEFAULT_SETTLE_DURATION = 300;
    public const DEFAULT_SSE_SEND_RETRY = 1000;
    public const DEFAULT_FRAGMENT_MERGE_MODE = FragmentMergeMode::Morph;
}
